source/features/pr-earliest-tag-after-merge.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+import oneTime from 'onetime';
+import TagIcon from 'octicon/tag.svg';
+import {observe} from 'selector-observer';
+import * as pageDetect from 'github-url-detection';
+
+import fetchDom from '../libs/fetch-dom';
+import features from '../libs/features';
+import {getRepoURL} from '../libs/utils';
+
+const getEarliestTag = cache.function(async (mergeCommit: string): Promise<string[] | undefined> => {
+	const firstAssociatedTag = await fetchDom<HTMLAnchorElement>(`/${getRepoURL()}/branch_commits/${mergeCommit}`, 'ul.branches-tag-list li:last-child a');
+	if (!firstAssociatedTag) {
+		return;
+	}
+
+	return [firstAssociatedTag.href, firstAssociatedTag.textContent!];
+}, {
+	cacheKey: ([mergeCommit]) => `earliest-tag:${getRepoURL()}/${mergeCommit}`
+});
+
+async function addEarliestTag(discussionHeader: Element) {
+	const mergeCommit = select('div.TimelineItem-body > a[href*="commit"] > code.link-gray-dark')!.textContent!;
+	const [href, textContent] = await getEarliestTag(mergeCommit) ?? [];
+
+	if (!href) {
+		return;
+	}
+
+	discussionHeader.append(
+		<span className="tooltipped tooltipped-s" aria-label={`${textContent} was the earliest tag after this PR was merged`}>
+			• <TagIcon width={14} className="d-inline-block mx-1 text-gray-light"/>
```

Comment:
`d-inline-block`? As far as I know SVG is already inline block. 
```suggestion
			• <TagIcon width={14} className="mx-1 text-gray-light"/>
```

Reply:
Was following the copy icon